src/Transition/OutputRulesFiles/OutputChanges8-5-0-to-8-6-0.md

```diff
@@ -12,4 +12,8 @@ This will eventually become a more structured file, but currently it isn't clear
 
 The Zone Summary table in the Input Verification and Results Summary Report always had a column labeled "Gross Wall Area" which was the above ground wall area for each zone. This change relabeled that column to be called "Above Ground Gross Wall Area" and added an entirely new column called "Underground Gross Wall Area" that shows the amount of underground wall area for each zone.
 
+### JtoKWH Unit Conversion Impacts LEED Summary Report
+
+When the JtoKHW unit conversion option is used in the OutputControl:Table:Style object it now changes the units used in the LEED Summary report. Columns that were previously left as GJ are now expressed as kWh and columns that were previously MJW/m2 are now expressed as kWh/m2.
+
```

Comment:
The JtoKWH option is not the default, doing no conversion is the default. When JtoKWH is chosen it currently just applies to the ABUPS and monthly reports, after this is merged it would also apply to the LEED Summary report.